ticket.php

```diff
@@ -485,14 +502,51 @@
                             <?php }
 
                             if (!empty($contact_phone)) { ?>
-                                <i class="fa fa-fw fa-phone text-secondary ml-1 mr-2 mb-2"></i><?php echo $contact_phone; ?>
+                                <i class="fa fa-fw fa-phone text-secondary ml-1 mr-2 mb-2"></i><a href="tel:<?php echo $contact_phone; ?>"><?php echo $contact_phone; ?></a>
                                 <br>
                             <?php }
 
                             if (!empty($contact_mobile)) { ?>
-                                <i class="fa fa-fw fa-mobile-alt text-secondary ml-1 mr-2 mb-2"></i><?php echo $contact_mobile; ?>
+                                 <i class="fa fa-fw fa-mobile-alt text-secondary ml-1 mr-2 mb-2"></i><a href="tel:<?php echo $contact_mobile; ?>"><?php echo $contact_mobile; ?></a>
                                 <br>
                             <?php } ?>
+                            
+                            <hr>
+                            
+                            
+                            
+                            <?php
+
+  							  	$sql_prev_ticket = "SELECT ticket_id, ticket_created_at, ticket_subject, ticket_status, ticket_assigned_to FROM tickets WHERE ticket_contact_id = $contact_id AND ticket_id < $ticket_id ORDER BY ticket_id DESC LIMIT 1";
+    							$row = mysqli_fetch_assoc(mysqli_query($mysqli, $sql_prev_ticket));
+
+   								 if ($row) {
+        							$prev_ticket_id = $row['ticket_id'];
+    								//    $prev_ticket_created_at = $row['ticket_created_at'];
+        							$prev_ticket_subject = $row['ticket_subject'];
+        							$prev_ticket_status = $row['ticket_status'];
+    								//    $prev_ticket_assigned_to = $row['ticket_assigned_to'];
```

Comment:
Could you please use intval (for the ID) and htmlentities (for the subject/status) when outputting onto the page? This helps prevent XSS attacks.

Like: $prev_ticket_subject = nullable_htmlentities($row['ticket_subject']);